### ISSUE TITLE
Guns that can't be crafted in the workbench

### ISSUE THREAD

**Issue by SkyLazy334** [Cubed-Development/Modern-Warfare-Cubed#132]
## Describe the bug

***Simple! The guns doesn't have recipes in the workbench - Don't know about their ammos and magazines***
***Here is a list of weapons that i searched using JEI and look for them in the workbench but doesn't have it***
- M249
- M40A6
- HK 417
- AR-15
- MAS-21
- PP-91 Kedr
- VSS Vintorez
- SVD Dragunov
- KS-23
- SV-98
- APC9
- S7-10 Tricun
- SA80
- Remington-700
- Browning Hi-power
- Colt M1911
- Colt Python
- Taurus Raging Hunter .44 magnum
- Chiappa Rhino
- S&W .500 Magnum
- MP-40
- M1928 Thompson
- AK-101
- AK-74
- AK-47
- AK-12
- FN F2000
- Steyr AUG A1
- Famas F1
- STG-44
- M1873
- Mare's leg
- G2 Contender
- Remington-870
- Browning Auto-5
- M1 Carbine
- M1 Garand
- G43 Gewehr
- M1903 Springfield
- Karabriner-98 Kurz
- Krag Jorgensen
- APS
- M712
- Uzi
- Saiga 12
- Spas-12
- MP-43E
- HS-12
- M1897
- G3
- Bren MkII
- M1941 Johnson (LMG)
- M1941 Johnson (Rifle)
- DP-28
- MG-34
- MG-42
- M60E4
- Stoner A1
- SSG-08
- DSR1
- L96A1
- M200 Intervention
- AS50
- GL-06
- M79 GL
- M32 MGL
- RPG-7
- M202 FLASH
- M134
- Chainsaw

## Reproducibility
Please try to reproduce the issue with as few other mods as possible. Then check one of the following checkboxes according to your results. To check a checkbox, replace the space between the square brackets with an x (like this [x]) or create the issue and check the checkbox by clicking on it.

- [x] I reproduced this issue with as few other mods as possible installed.
- [ ] I am unable to reproduce this issue consistently.

## To Reproduce

***Steps to reproduce the behavior:***

1. Go to 'weapon/armor workbench'
2. Click on the right arrow '>'
3. Go to the gun section
4. Search for any gun on the list above
5. You will see that it doesn't show in the workbench

## Expected behavior

***A clear and concise description of what you expected to happen.***

## Screenshots or/and videos

![image](https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/127067911/800ee316-2739-4178-a72e-48ff68528fab)

## Logs or/and crash reports

***No crash - Just missing recipes***

## Versions

- Modern Warfare Cubed: [Modern-Warfare-Cubed-0.1-Dev-11] 
- Forge: [1.12.2 - 14.23.5.2860]
- Java: [JDK 8 - 352-b08]
- Java JVM: [Hotspot]

## Specification:

- OS: [Windows 11]
- CPU: [13th Gen Intel(R) Core(TM) i5-13400F]
- GPU: [NVIDIA GeForce RTX 3060]
- RAM Allocated: [32 GB]

**Comment by Koud-Wind** [comment]
Are you currently playing LAN games?

**Comment by SkyLazy334** [comment]
> Are you currently playing LAN games?

No. As in the screenshot, it is single player world

**Comment by Desoroxxx** [comment]
Although not all of them should be craftable there is some that should definitely be

**Comment by SkyLazy334** [comment]
> 

Yeah, There are guns still craftable but it just limited the gameplay since there only a few gun to use over again

**Comment by Koud-Wind** [comment]
.minecraft\config\mwc\crafting
Check if this folder contains craftingmappings. json?

**Comment by Braiam23** [comment]
> .minecraft\config\mwc\crafting Check if this folder contains craftingmappings. json?

the craftingmappings. json contains the basic crafts like ak-47 but they disappear after adding more crafts of other weapons, in my opinion I think the workbench has some limitation of craftable weapons

**Comment by Koud-Wind** [comment]
> the craftingmappings. json contains the basic crafts like ak-47 but they disappear after adding more crafts of other weapons, 

Isn't this already fixed?

**Comment by funkymonkeyo** [comment]
I am having this issue as well (give or take a few weapons) on the same version of forge, except my mwc/crafting folder doesn't contain craftingmappings.json. How did this get fixed and/or how do I fix it?

**Comment by funkymonkeyo** [comment]
Also, I can't seem to craft cartridges of any caliber.
